src/openrct2/network/NetworkBase.h

```diff
@@ -12,10 +12,17 @@
 
 #ifndef DISABLE_NETWORK
 
+namespace OpenRCT2
+{
+    struct IContext;
+}
+
 class NetworkBase
 {
+    OpenRCT2::IContext& _context;
```

Comment:
On https://github.com/OpenRCT2/OpenRCT2/pull/15232/commits/40313b1ea3182adbb4586e9c8ca13032a19a0350 shouldn't this be down there along with the other variable declarations?

Reply:
This was there temporarily until I had the system base.